pkgs/build-support/docker/examples.nix

```diff
@@ -123,6 +123,14 @@ rec {
     finalImageTag = "2.2.1";
     finalImageName = "nix";
   };
+  # Same example again, but this time specify an os, an arch and a cpu variant
+  busyboxFromDockerHub = pkgsCross.armv7l-hf-multiplatform.pullImage {
+    imageName = "docker.io/library/busybox";
+    imageDigest = "sha256:ef320ff10026a50cf5f0213d35537ce0041ac1d96e9b7800bafd8bc9eff6c693";
+    sha256 = "sha256-9T6tP/xnZwaTH+qkbkVuXhCk+qZGE3WBY8T+Ug4qCmc=";
+    finalImageName = "docker.io/library/busybox";
+    finalImageTag = "stable-uclibc";
+  }
```

Comment:
Syntax error. I was willing to take a risk with you to simplify the function and potentially break other users' code, but how could I do that if I'm the only one who cares to run the tests? I mean a mistake is easily made, but this problem has been around since you opened the PR and clearly you haven't tried to test the code then or when iterating.
This signals that I'm going to have to be very strict about this PR, and I'll have to spend a lot more time on it to basically do your job. Frankly, that's not something I can free up time for. Maybe you would like to be a dockerTools maintainer? I could help with that.

Reply:
Hi, I'm really sorry about that, to be honest, I'm not able to easily run the tests right now and I am still a beginner, but I know it's not the good answer.
I took the time to re-run all the tests / examples and everything is working now except the cross-compilation test that fails, but if understand well according to your comment in the file, I don't have a compatible remote builder for the moment (I will try to do it in the near future), so everythings seems okay.
Sorry again 😖

Reply:
I'm sorry too. I shouldn't have assumed anything about you or your work.

>  I don't have a compatible remote builder

Maybe [`nix-community`](https://matrix.to/#/#nix-community:nixos.org) can help, or if you happen to have a different combination, you could change it temporarily to use two `system` values that you can build with a remote builder.

Reply:
After a bit of trouble configuring my remote builder, I could finally run all the tests successfully, even the cross-compilation one!